<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPasswordMangerDialog</class>
 <widget class="QDialog" name="QPasswordMangerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>276</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>378</width>
    <height>276</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>378</width>
    <height>318</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Host communication password manger</string>
  </property>
  <widget class="QGroupBox" name="groupBoxPasswordList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>256</height>
    </rect>
   </property>
   <property name="title">
    <string>Password List</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>186</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Password Info</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>65</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPwdAlias">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPwdOne">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPwdTwo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>125</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>175</y>
     <width>186</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation</string>
   </property>
   <widget class="QPushButton" name="pushButtonAddOnePwdItem">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemovePwdItem">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>55</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadPasswordList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>305</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Load List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSavePwdList">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>305</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save List</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditPwdAlias</tabstop>
  <tabstop>lineEditPwdOne</tabstop>
  <tabstop>lineEditPwdTwo</tabstop>
  <tabstop>pushButtonAddOnePwdItem</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonSavePwdList</tabstop>
  <tabstop>pushButtonLoadPasswordList</tabstop>
  <tabstop>pushButtonRemovePwdItem</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
